src/Commands/Branch/ListCommand.php

```diff
@@ -28,10 +28,10 @@ class ListCommand extends TerminusCommand implements SiteAwareInterface
      *   sha: SHA
      * @return RowsOfFields
      *
-     * @param string $site_id The name of the site to list the branches of
+     * @param string $site_id Site
      *
      * @usage terminus branch:list <site>
-     *    Lists the Git branches on the Pantheon remote associated with <site>
+     *    Displays list of Git branches within <site>'s Pantheon remote repository.
```

Comment:
`Displays a list of Git branches within <site>'s Pantheon remote repository.`
I just added an 'a' here.